.golangci.yaml

```diff
@@ -7,18 +7,55 @@ linters:
     - gocritic
     - gocyclo
     - goimports
-    - golint
+    - revive
     - gosec
     - gosimple
     - govet
     - ineffassign
     - misspell
     - nakedret
     - lll
+    - noctx
+    - rowserrcheck
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+    - depguard
+    - dogsled
+    - exportloopref
+    - goconst
+    - goprintffuncname
+    - nolintlint
+    - dupl
+    - gochecknoinits
+
+ # don't enable:
+ #  - golint        # deprecated
+ #  - whitespace
```

Comment:
why not? :)

Reply:
I had problem with empty lines we keep in the code, which delimits block of code, e.g.: we will remove empty lines here: [this](https://github.com/k8gb-io/k8gb/blob/master/controllers/gslb_controller.go#L128-L184)